dev/benchmarks/macrobenchmarks/lib/src/path_tessellation.dart

```diff
@@ -0,0 +1,341 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class PathTessellationPage extends StatefulWidget {
+  const PathTessellationPage({super.key});
+
+  @override
+  State<PathTessellationPage> createState() => _PathTessellationPageState();
+}
+
+class _PathTessellationPageState extends State<PathTessellationPage>
+    with SingleTickerProviderStateMixin {
+  late AnimationController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+    _controller =
+        AnimationController(vsync: this, lowerBound: 1.0, upperBound: 1.3);
+    _controller.addListener(() {
+      setState(() {});
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final double scale = _controller.value;
+    return SafeArea(
+      child: ColoredBox(
+        color: Colors.black,
+        child: Stack(
+          fit: StackFit.expand,
+          children: <Widget>[
+            ListView.builder(
+              key: const Key(
+                  'list_view'), // this key is used by the driver test,
+              itemBuilder: (BuildContext context, int index) {
+                return Container(
+                  margin: const EdgeInsets.all(1.0),
+                  decoration: BoxDecoration(
+                    color: Colors.white.withOpacity(0.2),
+                  ),
+                  child: IconRow(iconSize: (30 + 0.5 * (index % 10)) * scale),
+                );
+              },
+              itemCount: 200,
+              itemExtent: 50,
+            ),
+            Positioned(
+              top: 0,
+              left: 0,
+              right: 0,
+              child: Container(
+                color: Colors.black.withOpacity(0.7),
+                height: 100,
+                child: IconRow(iconSize: 50.0 * scale),
+              ),
+            ),
+            Positioned(
+              bottom: 0,
+              left: 0,
+              right: 0,
+              child: ColoredBox(
+                color: Colors.black.withOpacity(0.7),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.stretch,
+                  children: <Widget>[
+                    SizedBox(
+                      height: 100,
+                      child: IconRow(iconSize: 55.0 * scale),
+                    ),
+                    MaterialButton(
+                      textColor: Colors.white,
+                      key: const Key(
+                          'animate_button'), // this key is used by the driver test
+                      child: const Text('Animate'),
```

Comment:
Rather than have a button toggle between them, can we configure this via an argument passed to the PathTessellationPage widget constructor - and then have two separate routes for static and dynamic?

Reply:
Or leave the toggle, but let the route configure the default? I guess its fine - I just have driver test PTSD

Reply:
I don't mind making this two separate routes. I actually wanted to do it that way initially but I think other tests seem to have toggles so I wasn't sure if that's the preferred way.

Reply:
Yeah, that's fine!

Reply:
Given that the home page button is actually used to push the route (`runDriverTestForRoute`), it doesn't really seem to make sense to add a separate route without adding a separate home button.

So I not quite sure what you prefer. I can either make this two routes + two buttons at the home page, or one route + animate button (leave as it is now).